lib/index.js

```diff
@@ -81,7 +80,6 @@ var Client = module.exports = function (config) {
   config.headers = config.headers || {}
   this.config = config
   this.debug = Util.isTrue(config.debug)
-  this.Promise = config.Promise || config.promise || Promise
```

Comment:
Is it worth logging a deprecation warning if `config.Promise` or `config.promise` is passed?

Reply:
yes, good point, won't hurt